coderd/rbac/builtin.go

```diff
@@ -203,6 +242,12 @@ var (
 	// The first key is the actor role, the second is the roles they can assign.
 	//	map[actor_role][assign_role]<can_assign>
 	assignRoles = map[string]map[string]bool{
+		system: {
```

Comment:
Add a comment that system is explicitly NOT included in the list of built-in roles so that it can never be assigned to a user.